CHANGELOG.md

```diff
@@ -27,7 +27,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
   - Libfabric-aws: `libfabric-aws-1.19.0-1`
   - Rdma-core: `rdma-core-46.0-1`
   - Open MPI: `openmpi40-aws-4.1.6-1`
-- Upgrade GDRCopy to version 2.4.
+- Upgrade GDRCopy to version 2.4 in all supported OSes, but in Centos 7 where version 2.3.1 is used.
```

Comment:
Nitpick: change "but in" to "except for".

Reply:
done